src/app/qgsattributesforminitcode.cpp

```diff
@@ -29,9 +29,12 @@ QgsAttributesFormInitCode::QgsAttributesFormInitCode()
   mInitCodeSourceComboBox->addItem( tr( "Provide code in this dialog" ) );
   mInitCodeSourceComboBox->addItem( tr( "Load from the environment" ) );
 
-  connect( mInitCodeSourceComboBox, static_cast<void ( QComboBox::* )( int )>( &QComboBox::currentIndexChanged ), this, &QgsAttributesFormInitCode::mInitCodeSourceComboBox_currentIndexChanged );
-  connect( pbtnSelectInitFilePath, &QToolButton::clicked, this, &QgsAttributesFormInitCode::pbtnSelectInitFilePath_clicked );
+  QgsSettings settings;
+  mInitFileWidget->setDefaultRoot( settings.value( QStringLiteral( "style/lastInitFilePathDir" ), "." ).toString() );
+  mInitFileWidget->setDialogTitle( tr( "Select Python file" ) );
```

Comment:
File: first letter of dialog title in upper case (https://github.com/qgis/qgis3_UIX_discussion/issues/43)